domain/src/main/assets/13.json

```diff
@@ -0,0 +1,476 @@
+{
+  "exploration": {
```

Comment:
Can you also add an image region state to the prototype lesson with all interactions? Prefer the fractions image instead of solar system if possible.

I think we should keep that prototype exploration covering all of our supported interactions to simplify future end-to-end tests & testing certain whole-exploration flows.

Reply:
I can't add the fraction image interaction as it requires an actual image hosted with actual exploration id and actual image URL

Reply:
I thought we already had a lesson that made use of this. Can you check the multiplication & place values lessons to see if there are any examples of image region being used?

Reply:
I couldn't find anything related to image region selection.

Reply:
Hmm, I actually couldn't either. Let's just use solar system then--it's a larger state, but it's at least something that we can have in the prototype.

Reply:
We cannot map that state to our existing prototype because that would lead to a different exploration id

Reply:
Please file an issue to track resolving this. I think the correct solution is making sure that we can load all interactions for the test explorations locally without needing to download assets. We also need to support being able to load explorations from both a local dev server, and the remote Oppia Android development server, as well as production. In this case, please file an issue tracking this for the image region selection interaction.

Reply:
done https://github.com/oppia/oppia-android/issues/1528

Reply:
Thanks!